cmd/spire-server/cli/run/run.go

```diff
@@ -553,6 +556,18 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 		sc.CATTL = ttl
 	}
 
+	if c.Server.UseLegacyDownstreamX509CATTL != nil {
+		sc.UseLegacyDownstreamX509CATTL = *c.Server.UseLegacyDownstreamX509CATTL
+		if sc.UseLegacyDownstreamX509CATTL {
+			sc.Log.Info("Using legacy downstream X509 CA TTL calculation")
```

Comment:
may we add a warning instead? and suggest to stop using it since it will be removed in the future?

Reply:
I've added the deprecation warning.